File: Section Reports/Summary/DataFieldExpressions/C#/DataFieldExpressions/OrderDetail.cs
namespace GrapeCity.ActiveReports.Samples.DataFieldExpressions
{
	class OrderDetail
	{
		public OrderDetail(string productName, double unitPrice, int quantity)
		{
			this.productName = productName;
			this.unitPrice  = unitPrice;
			this.quantity = quantity;
		}
		public string productName;
		public double unitPrice;
		public int quantity;
	}
}

File: Section Reports/Summary/DataFieldExpressions/VB.NET/DataFieldExpressions/OrderDetail.vb
Imports System
Class OrderDetail
	Public Sub New(ByVal productName As String, ByVal unitPrice As Double, ByVal quantity As Integer)
		Me.productName = productName
		Me.unitPrice = unitPrice
		Me.quantity = quantity
	End Sub
	Public productName As String
	Public unitPrice As Double
	Public quantity As Integer
End Class
